app/views/admins/registrations/new.html.erb

```diff
@@ -0,0 +1,49 @@
+<% if admin_signed_in? %>
+
+  <% if current_admin.super_admin? %>
+
+    <h2>Sign up</h2>
+
+    <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+      <%= devise_error_messages! %>
+
+      <div class="field">
+        <%= f.label :email %><br />
+        <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+      </div>
+
+      <div class="field">
+        <%= f.label :password %>
+        <% if @minimum_password_length %>
+        <em>(<%= @minimum_password_length %> characters minimum)</em>
```

Comment:
this password length check could be handled in the admin model file

Reply:
at present it looks like the password length check is handled the same way with the user views and model, so I'd favor leaving this as is and staying consistent. but I can also see how handling it in the model would give certain advantages, ie, wouldn't have to repeat code when writing a view to change passwords. won't change anything now but open to doing so on further conversation